apps/studio/src/components/tableview/TableTable.vue

```diff
@@ -993,6 +999,15 @@ export default Vue.extend({
           value: cell.getValue()
         }
       })
+
+      let valueToSave = cell.getValue(0)
+
+      if (isArray) {
+        if (!_.isArray(valueToSave)) {
```

Comment:
These exceptions make my spidey sense tingle, I think we need to solve this in the editor